.github/workflows/build_publish.yml

```diff
@@ -5,17 +5,27 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
-
-env:
-  BAHMNI_VERSION: 0.94
   
 jobs:
-  build:
+  build_publish:
     runs-on: ubuntu-latest
+    env:
+      HELM_CHART_PATH: package/helm/
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Provision resources for dev
```

Comment:
Can we refactor the step name?

Reply:
of course yes... how did this name even got there? :D